frontend/src/app.js

```diff
@@ -58,10 +58,6 @@ const App = () => {
     localStorage.setItem("dark", !theme);
   }
 
-  useEffect(() => {
```

Comment:
If u remove the useEffect here, how will the component re-render after the theme change?

Reply:
@Kajol-Kumari I think in this case it doesn't perform any side effects or log messages based on theme changes, and if the component doesn't rely on these changes to trigger a re-render, we can safely remove the useEffect block. The component will still re-render when its state or props change.  What are your thoughts on this?